cmusatyalab/openface
# Tls cert bugfix

### What does this PR do?

This fix adds the required dependencies to the docker and pip configs to support TLS for the web demo. 

It also includes pre-generated self-signed certs that will expire in ten years and removes the call to create-certs.sh from install-deps.sh.  This avoids users having to actually create the certs unless the want to.

### Where should the reviewer start?

### How should this PR be tested?

Run the docker image (after it's been created). The demo webapp should start properly with no openssl related errors.  The docker url is now http**s**://docker_ip:8000 

I tested it on win10, I can test it again once the docker image is rebuilt.

### Any background context you want to provide?

Docker was missing a few required libraries to support TLS in python. Also the install-deps script which created the certs is not included in the docker config so it was never generating the certs. To avoid the extra complication of creating the certs, I just opted for including pre-generated long lived certs with the option of generating new ones (via create-cert.sh) 

### What are the relevant issues?

#261 

# Screenshots (if appropriate)

# Questions:

+ Do the docs need to be updated?

Yes (the url is now https not http)

+ Does this PR add new (Python) dependencies?

Yes it adds both OS package and python packages
